int32 localization_mode
